server/host/host.cpp

```diff
@@ -1,6 +1,4 @@
 #include <openenclave/host.h>
-#include <stdio.h>
-#include <iostream>
 
 #include "enclave.h"
```

Comment:
This file `enclave.h` seems to be empty. Perhaps an oversight? I believe that's where the `Enclave` class was defined.

Reply:
Oh interesting, looks like the code in that file wasn't pushed for whatever reason. Thanks for catching this!